src/utils.ts

```diff
@@ -63,3 +66,32 @@ export function getValueWithOptions<DataType>(
 
   return value as DataType;
 }
+
+export function serialize(valueToSerialize: unknown): string {
+  return JSON.stringify(valueToSerialize, (_, value) => {
+    if (typeof value === "bigint") {
+      return {
+        value: value.toString(),
+        [BIGINT_KEY]: BIGINT_BRAND,
```

Comment:
This is very clever mechanism! A little extension based on your approach can become more general. We don't need both BIGINT_KEY and BIGINT_BRAND to indicate it's a bigint, so, maybe it can be TYPE_KEY and TYPE?

Reply:
@ailisp Could you perhaps give an example or code snippet of what you are thinking about when you say _more general_?

Reply:
For example, if later we also want to support serialize and deserialize Date, we can extend your serialize code for date:
```js
return {
  value: value.toString(),
  [TYPE_KEY]: DATE_TYPE,
}
```

And in deserialization, we can always check `TYPE_KEY`, see what type it is and call bigint / date deserialization correspondingly

Reply:
@ailisp I added something similar along with an implementation of `Date` serialization and an accompanying test in [this commit](https://github.com/near/near-sdk-js/pull/228/commits/d843af664b8b1bdc70719b859a3715cf63d4a884)